weave/trace_server_bindings/remote_http_trace_server.py

```diff
@@ -290,11 +290,12 @@ def call_read(self, req: Union[tsi.CallReadReq, dict[str, Any]]) -> tsi.CallRead
     def calls_query(
         self, req: Union[tsi.CallsQueryReq, dict[str, Any]]
     ) -> tsi.CallsQueryRes:
-        return self._generic_request(
-            "/calls/query", req, tsi.CallsQueryReq, tsi.CallsQueryRes
-        )
+        # This previously called the deprecated /calls/query endpoint.
```

Comment:
I tried using the `@deprecated` decorator on this but it seems the default configuration of `warnings.warn` will not show it anyway if you're not calling it from main.